ranger/config/rifle.conf

```diff
@@ -70,6 +70,7 @@ ext x?html?, has chromium-browser, X, flag f = chromium-browser -- "$@"
 ext x?html?, has chromium,         X, flag f = chromium -- "$@"
 ext x?html?, has google-chrome,    X, flag f = google-chrome -- "$@"
 ext x?html?, has opera,            X, flag f = opera -- "$@"
+ext x?html?, has firefox-developer-edition, X, flag f = firefox-developer-edition -- "$@"
```

Comment:
I know we usually go by rarity. But I'm not sure betas and dev editions should follow the same rule. Opinions, @Vifon?

Reply:
I'd keep it out of the default rifle.conf. Users can and should add it manually in such cases.

Reply:
I'm not really opposed with having it in the default config. Do you think it'd be fine at the end of the list?

Reply:
Fair enough, I can see why you wouldn't want to clutter the default with loads of test/dev editions of software :+1:

Reply:
@swalladge, if you'll remove this line, I'll be happy to merge the PR. :)